### PR TITLE
Create EditProfile

### PR DESCRIPTION
**Pull request by hayleyw7**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Create populate, & display EditProfile component

## What (if anything) did you refactor?
-

## Were there any issues that arose?
- The defaultValues in the type drop downs are not yet working (we will want them to be the user's types if they have one set already)
- The handleSubmit function is just pseudocode for now

## Is there anything that you need from your teammates?
- Help with defaultValues?

## Any other comments, questions, or concerns?
- We will have to later add the handleSubmit function.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [ ] dev tools
- [ ] Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -29,8 +30,9 @@ function App() {
       <header className="App-header">
         <h1>EnneaLink</h1>
       </header>
-      {!loading && <Profile profileView={data.getUserStats} />}
-      <UserDisplay friends={friends} allUsers={allUsers} />
+      {!loading && <EditProfile /> }
```
Comment (MTCawthray) [inline src/components/App/App.js:34]: Once we get all of our components built out and Router incorporated we will be able to have a unified return in the App component and we should stop getting conflicts every time we send up a PR!

File: src/components/EditProfile/EditProfile.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import './EditProfile.css';
+
+class EditProfile extends Component {
```
Comment (MTCawthray) [inline src/components/EditProfile/EditProfile.js:4]: We will refactor this into a functional component. I'm wondering if we'll even need any hooks in here or if we can just send the data as a mutation!

File: src/components/EditProfile/EditProfile.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  constructor() {
+    super();
+
+    // do we need this with graphql?
+
+    // const {myersBrigg, enneagram} = user;
+
+    // this.state = {
```
Comment (MTCawthray) [inline src/components/EditProfile/EditProfile.js:12]: Like I said  above, we might not even need state! We'll have to play around with that.

File: src/components/App/App.js (hunk 1 above)
Comment (hayleyw7) [reply src/components/App/App.js:34]: Yes, definitely!

File: src/components/EditProfile/EditProfile.js (hunk 2 above)
Comment (hayleyw7) [reply src/components/EditProfile/EditProfile.js:4]: I'm thinking hooks? But we'll see!

File: src/components/EditProfile/EditProfile.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    e.preventDefault();
+
+    // save user type & set state in App to match inputs
+  }    
+
+  // maybe later add a cancel-btn or back-btn
+
+  render() {
+    return (
+      <form className='edit-profile-form'>
```
Comment (MTCawthray) [inline src/components/EditProfile/EditProfile.js:35]: Looks like a good skeleton to work with for our type form!

File: src/components/Profile/Profile.js [hunk 5]
```diff
@@ -16,7 +16,7 @@ const Profile = ({ profileView }) => {
         <h4 className="myers-briggs-name">{myersBrigg.name}</h4>
         <p>{myersBrigg.description}</p>
         <div className="link-container">
-          <a className="myers-briggs-link" src={myersBrigg.link}>Learn More
+          <a className="myers-briggs-link" href={myersBrigg.link}>Learn More
```
Comment (MTCawthray) [inline src/components/Profile/Profile.js:19]: good catch on the src=>href. That was my bad!

**Review by MTCawthray — APPROVED** [review]

File: src/components/EditProfile/EditProfile.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+          <option value='isfp'>ISFP</option>
+
+          <option value='estj'>ESTJ</option>
+          <option value='istj'>ISTJ</option>  
+          <option value='estp'>ESTP</option>
+          <option value='istp'>ISTP</option>
+        
+        </select>
+
+        <select name='type-list'>
```
Comment (nataliemcintyre2021) [inline src/components/EditProfile/EditProfile.js:64]: Drop-down options look great!

File: src/components/EditProfile/EditProfile.js (hunk 2 above)
Comment (nataliemcintyre2021) [reply src/components/EditProfile/EditProfile.js:4]: Agreed - hooks could be a good refactor!

**Review by nataliemcintyre2021 — APPROVED** [review]

**Pull request merged**